pkg/acsengine/const.go

```diff
@@ -29,6 +29,15 @@ const (
 	DefaultInternalLbStaticIPOffset = 10
 	// DefaultNetworkPolicy is disabling network policy enforcement
 	DefaultNetworkPolicy = "none"
+	// DefaultKubernetsDnsServiceIP specifies the IP address that kube-dns
+	// listens on by default. must by in the default Service CIDR range.
+	DefaultKubernetsDnsServiceIP = "10.0.0.10"
```

Comment:
There's a lot of `Kubernets` in here. (Missing the last `e`: `kubernets vs kubernetes`).